tests/application/job_service/test_job.py

```diff
@@ -14,17 +14,27 @@
 class TestJob:
 
     @pytest.mark.asyncio
-    async def test_if_service_raises_error_then_job_raises_error(self):
-        mock_apply_to_elster = MagicMock(side_effect=EricProcessNotSuccessful)
+    async def test_if_service_raises_raisable_error_then_job_raises_error(self):
+        mock_apply_to_elster = MagicMock(side_effect=EricProcessNotSuccessful(610201215))
         mock_service = MagicMock(apply_to_elster=mock_apply_to_elster)
 
         with pytest.raises(EricProcessNotSuccessful):
             await perform_job(request_id=uuid4(), repository=MagicMock(), service=mock_service,
                               payload_type=MagicMock(), logger=MagicMock())
 
+    @pytest.mark.asyncio
+    async def test_if_service_raises_not_raisable_error_then_job_raises_error(self):
```

Comment:
```suggestion
    async def test_if_service_raises_not_raisable_error_then_job_does_not_raise_error(self):
```

I can directly fix that